app/vminsert/main.go

```diff
@@ -220,6 +221,29 @@ func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 		}
 		w.WriteHeader(http.StatusOK)
 		return true
+	case "/api/v1/newrelic":
```

Comment:
should it be `/newrelic/api/v1` instead, as for all other protocols?

Reply:
update